src/luarocks/upload/api.lua

```diff
@@ -126,7 +135,7 @@ if not ltn12_ok then -- If not using LuaSocket and/or LuaSec...
 function Api:request(url, params, post_params)
    local vars = cfg.variables
    local json_ok, json = require_json()
-   if not json_ok then return nil, "A JSON library is required for this command." end
+   if not json_ok then return nil, "A JSON library is required for this command. "..json end
```

Comment:
this same log message is in multiple places.

I'll also note that this is a minor merge conflict with #466